src/pages/api/user/login.ts

```diff
@@ -19,22 +17,20 @@ export interface LoginResponseType {
 
 export default async function handler (
     req: NextApiRequest,
-    res: NextApiResponse<LoginResponseType>
+    res: NextApiResponse
 ) {
     const { email, password } = req.body;
 
     // Error: Not all fields are filled out.
-    if(!email || !password) {
-        res.status(400);
-
-        throw new Error("Please add all fields.");
+    if (!email || !password) {
+        res.status(400).json({ error: "Please fill out all fields." });
+        return;

```

Comment:
Would this not have the same effect as making it one line?

Reply:
I have to add the curly braces because if no curly braces then it executes the first command its given, which in this case is the res.status, and then the return would exist outside of the if block. So... I can do:
`if (something) { res.status().json(); return; }`
in one line if that's what you prefer. Plus, this isn't the most idea solution for returning errors. We still have to talk about that because I realized some faults with the ApiResponse class that's currently in lib